v0/util/error.js

```diff
@@ -31,7 +31,12 @@ function ErrorBuilder() {
   };
 
   this.setStatTags = arg => {
-    this.err.statTags = arg;
+    this.err.statTags = {
+      destType: arg.destType || arg.destination,
```

Comment:
Why still support `destination` even after we fix it across the repo?

Reply:
The fix that we have done is more through the calling methods rather than through actual destination's `transform.js` or `util.js`. Hence for now, we have continued to support the earlier semantic

Reply:
We don't have to do that. It'll only increase our backlog to clean up the `destination` tags later.

Let's fix them all now.

Reply:
For non-cdk destinations, this change seems to make sense
For the CDK destination, are you trying to suggest we upgrade the CDK version and clean the tags in CDK as well ?

Reply:
Good point. I believe there are no breaking changes with latest CDK lib except for some error message format. If that's the case, let's proceed, otherwise, forget I even brought this up, but ensure to create an task in our backlog.